engine/src/main/java/org/hibernate/search/engine/search/predicate/dsl/QueryStringPredicateFieldMoreStep.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.search.predicate.dsl;
+
+/**
+ * The step in a "query string" predicate definition where the query string to match can be set
+ * (see the superinterface {@link QueryStringPredicateMatchingStep}),
```

Comment:
Haha, see, sharing superinterfaces also has downsides :)

Anyway, I guess it's for the best, as it allows callers to write generic code.